### PR TITLE
About update

### PR DESCRIPTION
**Pull request by Feargh**
Added content from Jo. 

Didn't add the baseball graphic or gif as I couldn't find anything I liked. 

I would like the card title to be left aligned but can't seem to catch the same margin as the below h3 and p elements.
```
<CardTitle className="text-2xl font-bold p-6 max-w-6xl mx-auto p-6">
					Nice to meet you
</CardTitle>
```

### CODE REVIEW

**Comment by AlexPD93** [comment]
Is the card title "Nice to meet you"?

**Comment by AlexPD93** [comment]
It looks good!

File: app/about/page.tsx [hunk 1]
```diff
@@ -11,137 +12,121 @@ export const metadata: Metadata = {
 
 export default function About() {
 	return (
-		<div className="max-w-6xl mx-auto px-4 py-8">
-			<h1 className="text-3xl font-bold mb-8 text-gray-800">
-				Welcome to the norm
-			</h1>
+		<Card className="mb-8 bg-white shadow-lg rounded-lg overflow-hidden bg-opacity-30">
+			<CardHeader className="px-6 py-4">
+				<CardTitle className="text-2xl font-bold p-6 max-w-6xl mx-auto p-6">
```
Comment (AlexPD93) [inline app/about/page.tsx:17]: I've had a look on my laptop and if you get rid of mx-auto it moves the text away from the middle. Then I think it is a case of getting the margin to line up with the image

**Review by AlexPD93 — COMMENTED** [review]
On smaller screens the image looks odd. It stays aligned to the left for a bit too long

**Comment by Feargh** [comment]
about-mobile-border-inspect

@AlexPD93 I can't find the element that is causing this border?

**Comment by Feargh** [comment]
below 400 pixels
above 500 pixels

It seems to be an issue when under 400 pixels. This happens on the main page also so I don't think it's an issue I'm causing.

**Comment by AlexPD93** [comment]
Yeah I'm having the same problem with a screen that is less than 400 pixels. I think it is something to do with the body but it is a separate issue

**Review by AlexPD93 — APPROVED** [review]

**Pull request merged**